.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release to npm and gh-pages.
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: A job to execute release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install packages.
+        run: yarn install
+      - name: Set up project.
+        run: yarn build
+      - name: Run tests.
+        run: yarn test
+      - name: Release to NPM
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+          NPM_EMAIL: ${{secrets.NPM_EMAIL}}
+          NPM_USERNAME: ${{secrets.NPM_USERNAME}}
+        run: yarn release
+      - name: Set up git identity
+        run: git config --global user.name "instructure-ui-ci" && git config --global user.email "instructure-ui-ci@instructure.com"
+      - name: Remove __build__ folder
+        run: rm -rf packages/__docs__/__build__
+      - name: Create __build__ folder
+        run: mkdir packages/__docs__/__build__
+      - name: Checkout gh-pages
+        run: git checkout gh-pages
+      - name: Checkout master
+        run: git checkout master
```

Comment:
this has always the latest commit? dont we need a `git pull` here?

Reply:
The first operation clones the repo, so master should be up to date